cdnauth/debug.py

```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint, render_template
+
+from cdnauth.auth import login_required
+
+
+bp = Blueprint("debug", __name__, url_prefix="/debug")
+
+
+@bp.route("/")
+@login_required
+def item():
+    return render_template("debug.html")
```

Comment:
There is an interesting side effect the way this works. The cookie setting only happens in the `cdn` module instead of the `auth` module so while we can debug the session like this, we can't actually debug the cookies because they don't get set. I think eventually we should see if we can move the cookie setting logic into `auth`. That added enough complexity to not include that refactor in this initial version because you set cookies as part of the response which is not the domain if `auth` so I left it alone for now.